src/main/kotlin/com/gatecheck/gatecheck/api/RequestController.kt

```diff
@@ -0,0 +1,34 @@
+package com.gatecheck.gatecheck.api
+
+//import com.gatecheck.gatecheck.api.template.DefaultUserResponse
+import com.gatecheck.gatecheck.model.Request
+import com.gatecheck.gatecheck.service.RequestService
+import org.jetbrains.annotations.NotNull
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.*
+import java.util.*
+
+@RestController
+@RequestMapping("/api/v1/request")
+class RequestController @Autowired constructor(private val requestService:RequestService, private val userService: UserService){
```

Comment:
Between the variable name and type definition there should be a space. Most of there you can fix by running the refactor utility built in IntelliJ, CTRL + ALT + L
Same goes for space between arguments "user.id, messages, amount, index"